userauth_test.go

```diff
@@ -135,6 +135,17 @@ func TestUserTokenAuthenticator_ValidateSignature_RSA(t *testing.T) {
 	_, brokenToken2 := a.Authenticate(r)
 	assert.Equal(t, "broken token (claims are empty): map[]", brokenToken2.Error()[0:38])
 
+	// Bad issuer
+	basIss, err := helper.CreateRSAToken(prKeyParsed, "ES256", "JWT", helper.WrongTokenAlgClaims)
```

Comment:
```suggestion
	basIss, err := helper.CreateRSAToken(prKeyParsed, "RS256", "JWT", helper.WrongTokenAlgClaims)
```
wrong algorithm for this test block